packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts

```diff
@@ -21,5 +21,8 @@ export function getSDKWallets(dappConfig?: DappConfig) {
     sdkWallets.push(new DevTWallet() as any);
   }
 
+  // Add new SDK wallet plugins (ones that should be installed as packages) here:
```

Comment:
not completely sure this is needed

Reply:
I think it's optional, but I liked the idea of flagging where people should modify the code as it just helps re-assure people they're modifying the right part of the file :)